resources/styles.css

```diff
@@ -69,7 +76,8 @@
 }
 
 .suite-tab {
-    margin-top: 47px;
+    margin-top: 0px;
```

Comment:
Nice that we don't need the magic number in suite-tab and run-card anymore!  (they where a lot of problems)

Please, try to directly remove the css got suite-tab, run-card padding and border, and see if everything is still correct (we may not need these styles anymore)

Reply:
perfect :P